### ISSUE TITLE
Bug: Copy Current URL broken

### ISSUE THREAD

**Issue by hbridge** [jiahaog/nativefier#657]
### Description

Copy current URL seems to be broken for nativefier on mac.  This is problematic for me since I need to copy Asana URLs for tasks.

### Steps to reproduce issue

1. Create an app (I created Google Calendar and Asana)
2. File > Copy Current URL

*Expected*: URL is copied
*Actual*: Javascript error, URL is not copied

### Details

- Are you nativefying a *public* website?  
- Feature request? Have you looked at `nativefier --help` to see if an existing option could fit your needs? No
- Full nativefier command used to build your app: `nativefier -n "Google Calendar" --internal-urls "https://(accounts|calendar).google.com/*" "https://calendar.google.com/calendar/r"`
- Version of Nativefier (run `nativefier --version`): `7.6.4`
- Version of node.js (run `node --version`): `v8.6.0`
- OS: `Mac OS 10.13.5`
- Error message / stack trace (if any): A JavaScript error occurred in the main process

```
TypeError: Error processing argument at index 0, conversion failure from undefined
    at click (/Users/hbridge/Google Calendar-darwin-x64/Google Calendar.app/Contents/Resources/app/lib/webpack:/app/src/components/menu/menu.js:69:23)
    at MenuItem.click (/Users/hbridge/Google Calendar-darwin-x64/Google Calendar.app/Contents/Resources/electron.asar/browser/api/menu-item.js:55:9)
    at Function.executeCommand (/Users/hbridge/Google Calendar-darwin-x64/Google Calendar.app/Contents/Resources/electron.asar/browser/api/menu.js:29:13)
```

**Comment by bacongravy** [comment]
@hbridge thank you for the bug report!

This looks like a duplicate of issue #633, which is fixed by PR #634.  Unfortunately, there hasn't been a release of `nativefier` since those changes were merged.  The next release of `nativefier` should include the fix.

**Comment by ronjouch** [comment]
Thanks for the reminder everyone, going to push a release this weekend 🙂.
